### ISSUE TITLE
Great job but with sometime with some strange behaviors 

### ISSUE THREAD

**Issue by deepbeepmeep** [InternLM/InternLM-XComposer#78]
Thanks a lot ! Great job! Quite often the accuracy is very good.
However I have observed the following shortcomings :
- sometime the model starts repeating the same sentence and keeps repeating it
- follow up conversation after the first the question / answer is not as good : the model won’t be able to talk about something it didn’t mention in the first answer. It seems LLaVA 1.5 was better at that. More multi turn conversations for the next version would be helpful
- lots of hallucinations with pictures of landmarks. It is wrong quite often when presented with a famous landmark. I. would recommend to adapt the dataset so that if it isnt sure it should not put a name on it. 

Keep on the great work.

**Comment by deepbeepmeep** [comment]
I forgot to mention I was talking about the ShareGPT4V multimodal model

**Comment by LightDXY** [comment]
hi, thanks a lot for your valuable feedback, the current version ShareGPT4V-7B is mainly designed for a fair comparison, so the SFT part only replaces 23k detailed descriptions (3.5% of the 665k SFT data). Maybe with a stronger pre-training phase, the SFT data should also be adjusted accordingly, this would be an interesting topic for the following study.

**Comment by deepbeepmeep** [comment]
You have provided one more evidence better that increasing the quality of captions is the best way to improve multi modal language models. Thanks for that !
Please don’t stop half way and use for finetuning the full ShareGPT4V dataset or even better merge it with the LLaVA 1.5 dataset. 

What do you mean by « stronger pretraining phase » ? My understanding was that you used already the full ShareGPT4V-PT dataset. That being said it would definitely help to add to the pretraining / finetuning dataset GPT4-V multi turn dialogues to improve long term consistency. Indeed finetuning on long conversations obtained indirectly from GPT is certainly what makes the Vicuna models the best models for multi turns conversation.

**Comment by xiujiesong** [comment]
I also observed the repetition problem of ShareGPT4V when describing an image, so is there anyway to solve this problem? Thanks.

**Comment by myownskyW7** [comment]
@xiujiesong We have released the InternLM-XComposer2. It can perform detailed image descriptions very well. You may try it. BTW, ShareCaptioner 2.0 will come after the holiday of the Chinese New Year. https://github.com/InternLM/InternLM-XComposer?tab=readme-ov-file#model-zoo
